deploy.bat

```diff
@@ -0,0 +1,2 @@
+grunt vulcanize
```

Comment:
Let's put this into Gruntfile.js as a new task called 'deploy'. Deploy can have 'vulcanize' as a child task.